dist/src/main/resources/api/backup-management-api.yaml

```diff
@@ -17,6 +17,31 @@ servers:
         description: Management server schema
 paths:
   /:
+    post:
+      summary: Takes a backup
+      description: Start taking a backup with the given backupId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TakeBackupRequest'
+      responses:
+        '202':
+          description: A backup has been successfully scheduled
+          $ref: '#/components/responses/TakeBackupResponse'
+        '400':
+          description: Backup is not enabled or configured on the brokers
```

Comment:
:wrench: Based on the discussion, another case for 400 is invalid backup ids, e.g. strings or negative values